### PR TITLE
Move windows builds to Git mono-repo

### LINKED ISSUE #13: Move to git mirrors for builds
https://mail.openjdk.java.net/pipermail/jdk8u-dev/2021-December/014457.html announced JDK 8u's conversion to a mono-repository with a live git mirror. At this point it should be doable to produce upstream builds from git mirrors. We should switch to building from git mirrors by the time we do 8u332 EA builds (post January).

### PR DESCRIPTION
**Pull request by stooke**
This PR is the Windows side of #13.
This hasn't been tested in the Github infrastructure, but runs correctly locally.

### CODE REVIEW

File: build8.cmd [hunk 1]
```diff
@@ -74,96 +67,114 @@
...
+	
+	if defined FIX_HEADER_PATCH (
+		@echo *** Add temporary patch to be able to build on Windows
+		@echo on
+		pushd "%WORK_DIR%/%OJDK_SRC_DIR%/hotspot"
+		C:\cygwin64\bin\wget -O JDK-8272214.patch "https://cr.openjdk.java.net/~andrew/openjdk8/8248901-fail/webrev.01/hotspot.patch" || exit /b 1
+		unix2dos JDK-8272214.patch
+		%HG% import --no-commit JDK-8272214.patch || exit /b 1
+		popd
+	)
```
Comment (jerboaa) [inline build8.cmd]: We should drop this temp patch code as `JDK-8272214` has been fixed in 8u312 already.

**Review by jerboaa — COMMENTED** [review]
Looks mostly good. I believe we can drop the old build fix patch?

File: build8.cmd (hunk 1 above)
Comment (stooke) [reply build8.cmd]: Absolutely - I wasn't sure.  Done.

**Review by jerboaa — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**